src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,43 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+  const resultArray = [];
+  let tmpString = "";
+  let tmpStringLength = tmpString.length;
+  let count = 0
+  let duplicateCharacter = [];
+  let lowerString
 
+  if(!text) throw '' 
+      
+  if(typeof text !== 'string')  throw 'Please enter a valid string'
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.